{"name": "risk user -- @1.14 cases", "status": "passed", "steps": [{"name": "Given risk makes get request: /reconciliation/user_data/", "status": "passed", "start": 1611571318900, "stop": 1611571318938}, {"name": "Then check response of request: You do not have permission to perform this action. is False", "status": "passed", "start": 1611571318939, "stop": 1611571318939}], "parameters": [{"name": "url", "value": "/reconciliation/user_data/"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "False"}], "start": 1611571318899, "stop": 1611571318939, "uuid": "6ac9d532-a218-41ce-a7a7-fe197d6be50c", "historyId": "4361e83895bf2d6b085d01c690dedd16", "fullName": "features/II-Reconciliation/III-reconciliation_access.feature:risk user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}